src/pages/PullRequestPage/subroute/FilesChangedTab/FilesChanged/FilesChanged.jsx

```diff
@@ -57,14 +54,31 @@ function FilesChangedTab() {
     )
   }
 
-  if (hasImpactedFiles(data?.impactedFiles)) {
+  if (
+    data?.impactedFilesType === 'ImpactedFiles' &&
+    data?.impactedFiles.length > 0
+  ) {
     return (
       <div className="flex flex-col gap-2">
         <Table />
       </div>
     )
   }
 
+  if (data?.impactedFilesType === 'UnknownFlags') {
```

Comment:
Have we defined an enum w/ `UknownFlags`? Might be nice if we ever rename what UnknownFlags is

Reply:
There is not one no, but  I can go ahead and make one, I mostly was leaning on not as it was the only instance using it yet. I dont have strong feelings about it.